tests/fixtures/golden/component-button/button.scss

```
@@ -0,0 +1,92 @@
+
+  // Size variants (data-btn attribute)
+  &[data-btn~="xs"] { font-size: var(--btn-size-xs, 0.6875rem); }
+  &[data-btn~="sm"] { font-size: var(--btn-size-sm, 0.8125rem); }
+  &[data-btn~="lg"] { font-size: var(--btn-size-lg, 1.125rem); }
+  &[data-btn~="xl"] { font-size: var(--btn-size-xl, 1.25rem); }
+  &[data-btn~="block"] { width: 100%; display: flex; }
+
+  // Style variants (data-style attribute)
+  &[data-style="outline"] {
+    background: transparent;
+    border: 1px solid currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)